pairinteraction/CMakeLists.txt

```diff
@@ -113,10 +113,37 @@ set_target_properties(wignerSymbols PROPERTIES POSITION_INDEPENDENT_CODE ON)
 target_link_libraries(pireal PRIVATE wignerSymbols)
 target_link_libraries(picomplex PRIVATE wignerSymbols)
 
-# Include Eigen
+find_package(BLAS)
```

Comment:
Would it make sense to also set `EIGEN_USE_BLAS`, or is it already included with `EIGEN_USE_LAPACKE_STRICT`?

Reply:
`EIGEN_USE_BLAS` and `EIGEN_USE_LAPACKE_STRICT` are orthogonal. You should benchmark to see whether there is any speedup.

Reply:
A `run_benchmarks` target, as you have set up in #139, would be handy.  Can you add it to this pull request (just the target, without the CI integration)? Or to the master and then rebase the pull request?